{"1.0":{"info":{"author":"Plone Security Team","author_email":"security@plone.org","bugtrack_url":null,"classifiers":["Framework :: Plone","Framework :: Plone :: 4.0","Framework :: Plone :: 4.1","Framework :: Plone :: 4.2","Framework :: Plone :: 4.3","Framework :: Plone :: 5.0","Framework :: Plone :: 5.1","Programming Language :: Python","Programming Language :: Python :: 2.6","Programming Language :: Python :: 2.7"],"description_content_type":null,"docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/plone","keywords":"plone security hotfix patch","license":"GPL","maintainer":"","maintainer_email":"","name":"Products.PloneHotfix20171128","package_url":"https://pypi.org/project/Products.PloneHotfix20171128/","platform":"","project_url":"https://pypi.org/project/Products.PloneHotfix20171128/","project_urls":{"Homepage":"https://github.com/plone"},"provides_extra":null,"release_url":"https://pypi.org/project/Products.PloneHotfix20171128/1.0/","requires_dist":null,"requires_python":"","summary":"Various Plone hotfixes, 2017-11-28","version":"1.0","yanked":false,"yanked_reason":null},"last_serial":18472982,"urls":[{"comment_text":"","digests":{"blake2b_256":"fdea9e0248bbd9fb10fc9e671c90eb6b190c00fa0f9208858ee61d3019fc322a","md5":"baa3cc95cd297f9aeda387d0c8553292","sha256":"b4f310edfd2da1d99c9e26c80b5913604ea860f00f53aca4f93799d3d7ee3280"},"downloads":-1,"filename":"Products.PloneHotfix20171128-1.0.tar.gz","has_sig":false,"md5_digest":"baa3cc95cd297f9aeda387d0c8553292","packagetype":"sdist","python_version":"source","requires_python":null,"size":13107,"upload_time":"2017-11-28T15:02:16","upload_time_iso_8601":"2017-11-28T15:02:16.736609Z","url":"https://files.pythonhosted.org/packages/fd/ea/9e0248bbd9fb10fc9e671c90eb6b190c00fa0f9208858ee61d3019fc322a/Products.PloneHotfix20171128-1.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}